apps/src/music/player/ToneJSPlayer.ts

```diff
@@ -19,7 +19,7 @@ class ToneJSPlayer implements AudioPlayer {
   private samplers: {[instrument: string]: Sampler};
   private activePlayers: Source<SourceOptions>[];
   private currentPreview: {
-    id: string;
+    url: string;
```

Comment:
Thanks for following through with all these name updates!

Reply:
Sure thing.  I figured they should be done, but wanted to make sure the general approach was approved before making all of them. :)